### ISSUE TITLE
Difficult to tell apart active/inactive boost button

### ISSUE THREAD

**Issue by tazz4843** [ihateblueb/snowdrop#170]
**Description**
This is right on the line between feature request/bug so feel free to change it if it doesn't fit as such.

I boosted a post recently in a dark environment so my screen was dim, and to me with my specific vision it looked like tapping boost did nothing and I only noticed once I picked up my phone and actually felt the haptics that it did something. This isn't an issue if the screen is bright (for me) but that's not really a great workaround.

iOS has a setting for [Differentiate Without Color](https://support.apple.com/en-by/guide/iphone/iph3e2e1fb0/ios#iph53d0f7b25) that's supposed to make it easier for anyone with colorblindness to tell if some UI element is active or not. I have this on system-wide. Ice Cubes (if I remember correctly) would fill in the centre of the boost icon with a dot that way it was obvious that it was active.

The favourite button does not have this issue given it's a filled/empty star.

**Environment**
 - Device: iPhone 15 Pro Max
 - OS: iOS 26.6
 - App version: 0.0.6-alpha (8681b4d@master)

**Comment by ihateblueb** [comment]
will be in the next release, thank you
